.github/workflows/node.yml

```diff
@@ -38,6 +38,6 @@ jobs:
         # See https://prettier.io/docs/en/options.html#end-of-line
         if: ${{ runner.os != 'Windows' }}
       - name: Run unit tests
-        run: npm test --if-present
+        run: npm test --if-present -- --coverage
```

Comment:
What is the empty flag `--` for?

Reply:
It separates command line arguments for `npm` itself from command line arguments for the test script.